gulp/jscs.js

```diff
@@ -1,7 +1,11 @@
 var gulp = require('gulp');
+var jspaths = require('./util/jspaths');
 var jscs = require('gulp-jscs');
 
+var fs = require("fs");
+var config = fs.readFileSync("./node_modules/mofo-style/linters/.jscsrc");
```

Comment:
personally I would specify `encoding` for `fs.readFIleSync` so it returns a string, not a buffer.